src/js/cellProfile.js

```diff
@@ -100,10 +100,8 @@ cellProfile.prototype.removeStatusIcons = function (txtID) {
  */
 cellProfile.prototype.validateDisplayName = function (displayName, displayNameSpan,txtID) {
     var MINLENGTH = 1;
-    var MAXLENGTH = 128;
-    var letters = /^[一-龠ぁ-ゔ[ァ-ヴー々〆〤0-9a-zA-Z-_]+$/;
-    var specialchar = /^[-_]*$/;
-    var allowedLetters = /^[0-9a-zA-Z-_]+$/;
+    var MAXLENGTH = 256;
```

Comment:
Please fix the condition checking
from: lenDisplayName >= MAXLENGTH
to: lenDisplayName > MAXLENGTH